src/Jellyfin.Database/Jellyfin.Database.Providers.Sqlite/SqliteDatabaseProvider.cs

```diff
@@ -120,4 +126,33 @@ public Task RestoreBackupFast(string key, CancellationToken cancellationToken)
         File.Copy(backupFile, path, true);
         return Task.CompletedTask;
     }
+
+    /// <inheritdoc/>
+    public async Task PurgeDatabase(JellyfinDbContext dbContext, IEnumerable<string>? tableNames)
+    {
+        ArgumentNullException.ThrowIfNull(tableNames);
+
+        var deleteQueries = new List<string>();
+        foreach (var tableName in tableNames)
+        {
+            try
+            {
+                deleteQueries.Add($"DELETE FROM \"{tableName}\";");
+            }
+            catch (System.Exception ex)
+            {
+                System.Console.WriteLine(ex);
```

Comment:
Use a logger.. but how can this throw here?

Reply:
eh old dbg code